components/dropdown/dropdown-content-styles.js

```diff
@@ -174,7 +170,7 @@ export const dropdownContentStyles = css`
 
 	@media (prefers-reduced-motion: reduce) {
 		:host([opened]), :host([opened-above]) {
-			animation: none;
+			animation: none !important;
```

Comment:
Nit: now that legacy-Edge support is dropping, we could actually start inverting these so that we only apply the animation if `prefers-reduced-motion: no-preference` is true. Would avoid an `!important` here.